app/controllers/api/v0/service_plans_controller.rb

```diff
@@ -5,12 +5,14 @@ class ServicePlansController < ApplicationController
       include Api::V0::Mixins::ShowMixin
 
       def order
-        service_plan = model.find(order_params[:service_plan_id].to_i)
+        service_plan = model.find(params_for_order[:service_plan_id].to_i)
         task = Task.create!(:tenant => service_plan.tenant, :status => "started")
 
-        #TODO: Publish a message with service plan ordering messaging client and simply
-        # return the task id, let the messaging client handle updating the task.
-        order_service_plan_and_update_task(task, service_plan)
+        messaging_client.publish_message(
+          :service => "platform.topological-inventory.operations-openshift",
+          :message => "order_service",
```

Comment:
Can this be a little more structured?  Like `ServicePlan.order` to make it easier to process generally on the operations side?